geth/transactions/types.go

```diff
@@ -0,0 +1,75 @@
+package transactions
+
+import (
+	"bytes"
+	"context"
+	"errors"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+// errors
+var (
+	ErrInvalidSendTxArgs = errors.New("Transaction arguments are invalid (are both 'input' and 'data' fields used?)")
+)
+
+// Result is a JSON returned from transaction complete function (used internally)
+type Result struct {
+	Hash  common.Hash
+	Error error
+}
+
+// RawDiscardResult is list of results from CompleteTransactions() (used internally)
+type RawDiscardResult struct {
```

Comment:
Is this type needed for anything?

Reply:
Used as return type on the [backend](https://github.com/status-im/status-go/blob/0e05743109560ce60b8b95ccad80669a84b55be3/geth/api/backend.go#L259) and [api](https://github.com/status-im/status-go/blob/0e05743109560ce60b8b95ccad80669a84b55be3/geth/api/api.go#L169)

Reply:
Yeah, but it looks like we can replace it with `error` and it will work the same, right?

Reply:
yep, you're right :)